src/app/api/update-quiz-config/route.ts

```diff
@@ -0,0 +1,41 @@
+import {updateQuizDataService} from '@/lib/core/service'
+import {getServerSession} from 'next-auth'
+import {NextResponse} from 'next/server'
+import {authOptions} from '../auth/[...nextauth]/route'
+
+export async function POST(request: Request) {
```

Comment:
The API URL could be something like this:

POST /api/quiz-config


REST API is about manipulating entity, the entities are quiz, assessment, quiz-config and etc.